src/RuleSet/Sets/SymfonySet.php

```diff
@@ -140,6 +140,13 @@ public function getRules(): array
             'phpdoc_no_alias_tag' => true,
             'phpdoc_no_package' => true,
             'phpdoc_no_useless_inheritdoc' => true,
+            'phpdoc_order' => [
+                'order' => [
```

Comment:
I hope this change was aligned with Sf community?

Reply:
Not exactly ;-) How can I reach them?

Reply:
ping @nicolas-grekas or @derrabus Is this change OK or should we revert it?

Reply:
LGTM